svc/kube_list_jobs.go

```diff
@@ -145,14 +135,15 @@ func (k *Kube) ListJobs(ctx context.Context, in *ListJobsInput) (out *ListJobsOu
 			item.ActiveAt = job.Status.StartTime.Local()
 		}
 
-		d, ok := inputs[job.Name]
-		if ok {
-			item.Input = d
-		}
-
-		d, ok = outputs[job.Name]
-		if ok {
-			item.Output = d
+		for _, dataset := range job.Spec.Template.Spec.Volumes {
+			if dataset.FlexVolume != nil {
+				if dataset.FlexVolume.Options["input/dataset"] != "" {
```

Comment:
Hey could you change this to the form in which it won't cause a panic if the option doesn't exist? I see this going wrong in the future when the options map changes around. So like:

```
if _, ok :=  dataset.FlexVolume.Options["input/dataset"]; ok {

}
```

Reply:
Ok, after running this https://play.golang.org/p/8B0qyBUn5uD it seems that this won't panic. I don't know why I though this. So nvm, it can be merged